### DISCUSSION TITLE
Certificate verification error

### DISCUSSION THREAD

**Discussion by duz52** [mitmproxy/mitmproxy#5033] [Q&A]
192.168.0.106:37204: Certificate verification error for xxx.xxx.com: self signed certificate (errno: 18, depth: 0)
192.168.0.106:37204: Ignoring server verification error, continuing with connection

**Comment by mhils** [comment]
I'm not sure what your question is? Mitmproxy assesses the certificate for `xxx.xxx.com` to not be valid, but since you presumably enabled `ssl_insecure`, it continues with the connection nonetheless and only emits a warning.
